lib/generators/alchemy/solidus/install/install_generator.rb

```diff
@@ -1,5 +1,12 @@
 require 'rails/generators'
-require 'rails/generators/alchemy/install/install_generator'
+require 'alchemy/version'
+
+if Alchemy.gem_version >= Gem::Version.new("5.0.0a")
```

Comment:
I think [it has to be](https://github.com/AlchemyCMS/alchemy_cms/blob/master/lib/alchemy/version.rb#L4) '5.0.0.a'